api/src/main/java/com/michelin/ns4kafka/services/schema/client/entities/SchemaCompatibilityRequest.java

```diff
@@ -0,0 +1,9 @@
+package com.michelin.ns4kafka.services.schema.client.entities;
+
+import lombok.*;
+
+@Getter
+@Builder
+public class SchemaCompatibilityRequest {
+    private String compatibility;
+}
```

Comment:
Why do we need this class ? Micronaut @Body String enough ?

Reply:
The Schema Registry requires a JSON object as body : https://docs.confluent.io/platform/current/schema-registry/develop/api.html#put--config-(string-%20subject) with a "compatibility" field :

{
  "compatibility": "FULL"
}

A simple map (e.g., <"compatibility", "FULL">) can be used as body, in place of the custom SchemaCompatibilityRequest object.

Reply:
Fixed, `SchemaCompatibilityRequest` replaced by single body String